storm-core/src/jvm/org/apache/storm/trident/Stream.java

```diff
@@ -124,6 +124,31 @@ public Stream parallelismHint(int hint) {
     }
 
     /**
+     * Sets the CPU Load resource for the current node
```

Comment:
minor nit: You may want to say this configuration is for the current operation instead of Node. Node representation is internal as you have mentioned in other comment. 

"Sets the CPU Load resource for the current operation"